google-cloud-pubsub/lib/google/cloud/pubsub/received_message.rb

```diff
@@ -179,9 +181,9 @@ def published_at
         #   # Shut down the subscriber when ready to stop receiving messages.
         #   subscriber.stop!
         #
-        def acknowledge!
+        def acknowledge! &block
```

Comment:
We do need to document what the block is for, when it is called, and what is passed to it.